cling/src/cmake/modules/RootMacros.cmake

```diff
@@ -909,6 +909,273 @@ function(ROOT_MODULE_LIBRARY library)
                              ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT libraries)
 endfunction()
 
+#----------------------------------------------------------------------------
+# function ROOT_ADD_TEST( <name> COMMAND cmd [arg1... ]
```

Comment:
Why do we need this? Is it because of `-Dclingtest=On`? If so, can you try adding `-Dtesting=Off`

Reply:
It still requires `ROOT_ADD_TEST`